vulnfeeds/vulns/vulns.go

```diff
@@ -320,14 +360,14 @@ func (v *Vulnerability) ToYAML(w io.Writer) error {
 	return encoder.Encode(v)
 }
 
-func timestampToRFC3339(timestamp string) (string, error) {
-	t, err := cves.ParseTimestamp(timestamp)
-	if err != nil {
-		return "", err
-	}
+// func timestampToRFC3339(timestamp string) (string, error) {
```

Comment:
Can we remove this completely?

Reply:
Yes, I missed this one when I was cleaning up, thanks.